src/App.tsx

```diff
@@ -1,14 +1,65 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { FilterTodos } from './types/FilterTodos';
+
+type FilterObject = {
+  query: string;
+  filterParameter?: FilterTodos | string;
+};
+
+const prepareTodos = (todos: Todo[], { query, filterParameter }: FilterObject) => {
+  let todosCopy = [...todos];
+
+  if (query) {
+    todosCopy = todosCopy.filter(todo => {
+      return todo.title.toLowerCase().includes(query.trim().toLowerCase());
+    });
+  }
+
+  if (filterParameter !== FilterTodos.All) {
+    todosCopy = todosCopy.filter(todo => {
+      switch (true) {
```

Comment:
You can pass filterParameter to stich and use FilterTodos.Completed and FilterTodos.Active for cases